Not using fixed utility path, but respect $PATH

10/09/2010 - created by Qing He <qing.he@intel.com>

diff --git a/m4/package_utilies.m4 b/m4/package_utilies.m4
index 165db35..0afe578 100644
--- a/m4/package_utilies.m4
+++ b/m4/package_utilies.m4
@@ -37,22 +37,22 @@ AC_DEFUN([AC_PACKAGE_UTILITIES],
     AC_PACKAGE_NEED_UTILITY($1, "$cc", cc, [C compiler])
 
     if test -z "$MAKE"; then
-        AC_PATH_PROG(MAKE, gmake,, /usr/bin:/usr/local/bin:/usr/freeware/bin)
+        AC_PATH_PROG(MAKE, gmake)
     fi
     if test -z "$MAKE"; then
-        AC_PATH_PROG(MAKE, make,, /usr/bin)
+        AC_PATH_PROG(MAKE, make)
     fi
     make=$MAKE
     AC_SUBST(make)
     AC_PACKAGE_NEED_UTILITY($1, "$make", make, [GNU make])
 
     if test -z "$TAR"; then
-        AC_PATH_PROG(TAR, tar,, /usr/freeware/bin:/bin:/usr/local/bin:/usr/bin)
+        AC_PATH_PROG(TAR, tar)
     fi
     tar=$TAR
     AC_SUBST(tar)
     if test -z "$ZIP"; then
-        AC_PATH_PROG(ZIP, gzip,, /bin:/usr/bin:/usr/local/bin:/usr/freeware/bin)
+        AC_PATH_PROG(ZIP, gzip)
     fi
 
     zip=$ZIP
@@ -65,25 +65,25 @@ AC_DEFUN([AC_PACKAGE_UTILITIES],
     AC_SUBST(makedepend)
 
     if test -z "$AWK"; then
-        AC_PATH_PROG(AWK, awk,, /bin:/usr/bin)
+        AC_PATH_PROG(AWK, awk)
     fi
     awk=$AWK
     AC_SUBST(awk)
 
     if test -z "$SED"; then
-        AC_PATH_PROG(SED, sed,, /bin:/usr/bin)
+        AC_PATH_PROG(SED, sed)
     fi
     sed=$SED
     AC_SUBST(sed)
 
     if test -z "$ECHO"; then
-        AC_PATH_PROG(ECHO, echo,, /bin:/usr/bin)
+        AC_PATH_PROG(ECHO, echo)
     fi
     echo=$ECHO
     AC_SUBST(echo)
 
     if test -z "$SORT"; then
-        AC_PATH_PROG(SORT, sort,, /bin:/usr/bin)
+        AC_PATH_PROG(SORT, sort)
     fi
     sort=$SORT
     AC_SUBST(sort)
@@ -93,21 +93,21 @@ AC_DEFUN([AC_PACKAGE_UTILITIES],
 
     if test "$enable_gettext" = yes; then
         if test -z "$MSGFMT"; then
-                AC_PATH_PROG(MSGFMT, msgfmt,, /usr/bin:/usr/local/bin:/usr/freeware/bin)
+                AC_PATH_PROG(MSGFMT, msgfmt)
         fi
         msgfmt=$MSGFMT
         AC_SUBST(msgfmt)
         AC_PACKAGE_NEED_UTILITY($1, "$msgfmt", msgfmt, gettext)
 
         if test -z "$MSGMERGE"; then
-                AC_PATH_PROG(MSGMERGE, msgmerge,, /usr/bin:/usr/local/bin:/usr/freeware/bin)
+                AC_PATH_PROG(MSGMERGE, msgmerge)
         fi
         msgmerge=$MSGMERGE
         AC_SUBST(msgmerge)
         AC_PACKAGE_NEED_UTILITY($1, "$msgmerge", msgmerge, gettext)
 
         if test -z "$XGETTEXT"; then
-                AC_PATH_PROG(XGETTEXT, xgettext,, /usr/bin:/usr/local/bin:/usr/freeware/bin)
+                AC_PATH_PROG(XGETTEXT, xgettext)
         fi
         xgettext=$XGETTEXT
         AC_SUBST(xgettext)
@@ -117,7 +117,7 @@ AC_DEFUN([AC_PACKAGE_UTILITIES],
     fi
 
     if test -z "$RPM"; then
-        AC_PATH_PROG(RPM, rpm,, /bin:/usr/bin:/usr/freeware/bin)
+        AC_PATH_PROG(RPM, rpm)
     fi
     rpm=$RPM
     AC_SUBST(rpm)
